operator/v1/types.go

```diff
@@ -59,10 +58,6 @@ type OperatorSpec struct {
 	// +optional
 	OperatorLogLevel LogLevel `json:"operatorLogLevel"`
 
-	// operandSpecs provide customization for functional units within the component
-	// +optional
-	OperandSpecs []OperandSpec `json:"operandSpecs,omitempty"`
```

Comment:
I assume we simply do not want support for applying a patch on the operand's deployment/pod?

Reply:
> I assume we simply do not want support for applying a patch on the operand's deployment/pod?

We don't today.  We'll see what we need later